modules/ROOT/pages/bundling-content-css.adoc

```diff
@@ -19,7 +19,7 @@ Example syntax for including the example content CSS in a bundle:
 |===
 |Module Syntax |Source |Example
 
-.2+|ES6+
+.2+|ES6+ (Webpack/Rollup)
```

Comment:
Why do we want to mention `Webpack/rollup`? And while other demos recommend to use Vite